CODEOWNERS

```diff
@@ -1 +0,0 @@
-* @operator-framework/combo-maintainers
```

Comment:
why was this removed?

Reply:
Looks like an oversight. I'll add it back.